src/envs/cover.py

```diff
@@ -345,3 +345,372 @@ def __init__(self) -> None:
     def types(self) -> Set[Type]:
         return {self._block_type, self._block_type_derived,
                 self._target_type, self._robot_type}
+
+
+class CoverMultistepOptions(CoverEnvTypedOptions):
+    """Cover domain with a lower level action space.
+
+    Useful for using and learning multistep options.
+
+    The action space is (dx, dy, dgrip). The last dimension
+    controls the gripper "magnet" or "vacuum".
+
+    The state space also needs to change to track x, y, grip.
+
+    The notion of allowed hand regions needs to be adjusted.
+    We will allow the gripper to move anywhere, except when
+    the y dimension is less than a certain threshold, then
+    the x dimension will still need to be within a hand region.
+
+    Currently simulate disallows dropping blocks except when
```

Comment:
Is this a current limitation, or an old comment that should be deleted?

Reply:
this is current. Ashay is working on addressing this limitation, but we'll do that in a separate PR